### PR TITLE
Creating object script added

### PR DESCRIPTION
**Pull request by avishekjana**

### CODE REVIEW

File: numbers-booleans-objects/creating-and-working-with-objects/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Script
+
+In this lesson, we will learn about Javascript Objects. 
+In Javascript, an object is something which is very powerful, and you are going to work with them all the time. So, let's get started.
```
Comment (mjzac) [inline numbers-booleans-objects/creating-and-working-with-objects/README.md:4]:
```suggestion
In JavaScript, an object is a powerful construct, and you are going to work with them all the time. So, let's get started.
```

**Review by mjzac — CHANGES_REQUESTED** [review]

File: numbers-booleans-objects/creating-and-working-with-objects/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+### Overview 
+A Javascript object is a collection of named values. Like many other programming languages, it can be compared to objects in real life. Compare it with a cup, for example. A cup is an object, with properties. A cup has a color, a design, weight, a material it is made of. In the same way, JavaScript objects can have properties, which define their characteristics.
```
[the remark this answers is not in the document]
Comment (avishekjana) [reply numbers-booleans-objects/creating-and-working-with-objects/README.md]: Resolved.

File: numbers-booleans-objects/creating-and-working-with-objects/README.md (hunk 1 above)
Comment (avishekjana) [reply numbers-booleans-objects/creating-and-working-with-objects/README.md]: Resolved.

File: numbers-booleans-objects/creating-and-working-with-objects/README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+# Script
+
+In this lesson, we will learn about Javascript Objects.
```
[the remark this answers is not in the document]
Comment (avishekjana) [reply numbers-booleans-objects/creating-and-working-with-objects/README.md]: Resolved.

**Pull request merged**